portfolio/src/main/webapp/blog.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8" name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,600,800&display=swap" rel="stylesheet">
+        <title>pear.</title>
+        <link rel="stylesheet" href="styleProjects.css">
+        <script src="scriptProject.js">
+        </script>
+    </head>
+    <body style="background-color:white" onload="getComments()">
```

Comment:
instead of hooking up js handlers in html, can you do so programmatically? (See: "DOMContentLoaded")